Dockerfile

```diff
@@ -46,6 +46,6 @@ CMD ["node", "dist/app.js"]
 
 ## Labels
 LABEL name="artemiscloud/activemq-artemis-jolokia-api-server"
-LABEL description="ActiveMQ Artemis Jolokia api-server"
+LABEL description="ActiveMQ Artemis Jolokia Api Server"
 LABEL maintainer="Howard Gao <hgao@redhat.com>"
-LABEL version="0.1.1"
+LABEL version="0.1.1"
```

Comment:
can you avoid changing the last empty line of the file please?

Reply:
sure